### PR TITLE
feat: print end status in watch mode

### PR DESCRIPTION
**Pull request by nnecec**
I sometimes wonder if I've built done in watch mode, because it print nothing when built sucessfully.
 
Do we need to print some hint like these in watch mode?

```bash
> bunchee -w

Watching project /Users/nnecec/Github/bunchee...
📦 Build in: 1.956s
📦 Build in: 595.334ms
src/bundle.ts(25,3): error TS6133: 'getPackageMeta' is declared but its value is never read.
src/bundle.ts(70,64): error TS2304: Cannot find name 'pkg'.
src/bundle.ts(71,38): error TS2304: Cannot find name 'pkg'.
src/bundle.ts(73,38): error TS2304: Cannot find name 'pkg'.
src/bundle.ts(75,26): error TS2304: Cannot find name 'pkg'.
src/bundle.ts(153,40): error TS2304: Cannot find name 'pkg'.
src/bundle.ts(177,5): error TS18004: No value exists in scope for the shorthand property 'pkg'. Either declare one or provide an initializer.

📦 Build in: 652.215ms
📦 Build in: 512.592ms
```

### CODE REVIEW

**Comment by nnecec** [comment]
Ignore this feat if you don't think it's necessary, just my personal opinion.

**Review by huozhi — COMMENTED** [review]
Thanks, this is a nice feature. 
Can we polish the output to sth like

```
✓ Built in xxx ms
```

Without including the emoji and `:`

**Comment by nnecec** [comment]
Now it look like

```bash
> bunchee -w
Watching assets in /Users/nnecec/Github/bunchee...
✓ Build in 2875.94ms
src/bundle.ts(30,3): error TS6133: 'formatDuration' is declared but its value is never read.

✓ Build in 1541.16ms
✓ Build in 1727.21ms
```

**Review by huozhi — COMMENTED** [review]
Looks good, can we add a test? Then I will merge later release this in a minor version. Thank you!

File: test/cli.test.ts [hunk 1]
```diff
@@ -294,6 +320,11 @@ describe('cli', () => {
       let stdout = ''
       ps.stdout?.on('data', (chunk) => (stdout += chunk.toString()))
       ps.stderr?.on('data', (chunk) => (stderr += chunk.toString()))
+      if (typeof timeoutClose === 'number') {
```
Comment (nnecec) [inline test/cli.test.ts:323]: Is this a correct solution?

File: test/cli.test.ts [hunk 2]
```diff
@@ -254,11 +256,35 @@ const testCases: {
       ]
     },
   },
+  {
+    name: 'output-in-watch',
+    dist: createTempDir,
+    timeoutClose: 4000,
+    expectedCode: 143,
```
[the remark this answers is not in the document]
Comment (huozhi) [reply test/cli.test.ts:263]: Sounds good, would be great to add some comments to explain it’s for SIGTERM

File: test/cli.test.ts (hunk 1 above)
Comment (nnecec) [reply test/cli.test.ts:323]: I think jest's timeout can't be used. If we don't kill the ps manually, watch mode will waiting at here

```ts
const code = await new Promise((resolve) => {
  ps.on('close', resolve)
})
```

test will always failed with timeout.

**Review by huozhi — APPROVED** [review]

**Comment by huozhi** [comment]
I refactored a bit the code, to leverage the build jobs count instead of using another counter, since that one could be incorrect that it might reaches to 0 ealier

**Pull request merged**